src/init.cpp

```diff
@@ -904,9 +904,9 @@ bool AppInitParameterInteraction()
         InitWarning(strprintf(_("Reducing -maxconnections from %d to %d, because of system limitations."), nUserMaxConnections, nMaxConnections));
 
     // ********************************************************* Step 3: parameter-to-internal-flags
-    if (mapMultiArgs.count("-debug") > 0) {
+    if (argsGlobal.IsArgSet("-debug")) {
         // Special-case: if -debug=0/-nodebug is set, turn off debugging messages
-        const std::vector<std::string>& categories = mapMultiArgs.at("-debug");
+        const std::vector<std::string>& categories = argsGlobal.ArgsAt("-debug");
```

Comment:
At the same time I didn't do it on purpose here. I'm happy to remove the "&" if that's preferred.